pkg/cmd/cli/cmd/deploy.go

```diff
@@ -352,6 +361,9 @@ func (o DeployOptions) cancel(config *deployapi.DeploymentConfig, out io.Writer)
 
 // reenableTriggers enables all image triggers and then persists config.
 func (o DeployOptions) reenableTriggers(config *deployapi.DeploymentConfig, out io.Writer) error {
+	if config.Spec.Paused {
+		return fmt.Errorf("cannot enable triggers for a paused deployment config")
```

Comment:
This isn't enforced on the server and should be benign anyway (since triggering shouldn't have an effect when paused). Seems safe to let users continue to mutate their spec.